.github/workflows/python-build-and-test.yml

```diff
@@ -19,3 +19,33 @@ jobs:
     - name: Test with pytest
       run: |
         python3 -m pytest quarkchain/cluster/tests/test_cluster.py
+    - name: Run py.test tests excluding integration test
+      run: |
+        python3 -m pytest --ignore quarkchain/cluster/tests/test_cluster.py
+    - name: Run py.test tests only on integration test
+      run: |
+        python3 -m pytest quarkchain/cluster/tests/test_cluster.py
+    - name: Run EVM tests
+      run: |
+        python3 quarkchain/evm/tests/test_state.py
+    - name: Download DB snapshot
```

Comment:
as I said earlier, we don't need check-db workflow, from line 31 to end of the file

Reply:
Done.